src/runtime_src/driver/zynq/drm/zocl/sched_exec.c

```diff
@@ -546,8 +549,9 @@ configure(struct sched_cmd *cmd)
 
 	for (i  = 0; i < exec->num_cus; i++) {
 		exec->cu_addr_phy[i] = cfg->data[i];
-		SCHED_DEBUG("++ configure cu(%d) at 0x%x\n", i,
-		    exec->cu_addr_phy[i]);
+		exec->cu_addr_virt[i] = ioremap(exec->cu_addr_phy[i], 0x80000);
```

Comment:
While this should be 0x10000, it seems better we create a micro than use instant number (if the CU size is not configurable any more)
Another thought is we'd better check the return value of ioremap to avoid bad CU physical address which can be anything by customer ert configuration command